### DISCUSSION TITLE
Does it support web publishing?

### DISCUSSION THREAD

**Discussion by AnkoGo123** [o3de/o3de#2069] [General]
Does it support web publishing?

**Comment by micronAMZN** [comment]
Assuming you are asking about WebAssembly or WebGL. No, O3DE does not currently support either. This is a topic that is being discussed on Discord.

**Reply by Akarian** [reply]
Hello, I can't find any news on this point. Is it still under discussion, implemented or will it never be realized ?

**Reply by micronAMZN** [reply]
When I initially replied to this discussion, there were several contributors who were interested in WebAssembly specifically. I'm not aware of any recent discussion around web publishing. There is no opposition adding support for this, however, as this is an open source project, it is up to contributors to drive its development.

**Comment by rgba16f** [comment]
The main reason web publishing hasn't been pursued any further is that O3DE doesn't support OpenGL/WebGL for rendering. Any web publishing support would need to wait for  WebGPU to become widely supported, and then a rendering backend would need to be created.

**Reply by mafster** [reply]
WebGPU is now here! Bumping this. Does anyone know of any updates on this front?

**Reply by adrian-antoci** [reply]
Any updates? Would be really interesting to see it running on web.

**Reply by i12345** [reply]
[PlayCanvas](https://github.com/playcanvas/engine) is one WebGL/WebGPU game engine.

[Here](https://github.com/i12345/playcanvas-extended) is my extended version of playcanvas.

I would enjoy being able to export from O3DE to the web for a single authoring repo.

**Comment by orkutmuratyilmaz** [comment]
any updates?
